### ISSUE TITLE
node v20.x is not supported.

### ISSUE THREAD

**Issue by dooglegh** [mancku/strapi-plugin-schemas-to-ts#34]
```
yarn develop
...
error strapi-plugin-schemas-to-ts@1.1.10: The engine "node" is incompatible with this module. Expected version ">=14.19.1 <=18.x.x". Got "20.9.0"
```

An error occurs when running in the installation environment with node v20.9.0.
node v20 is not supported.

**Comment by creazy231** [comment]
You should be able to use it with the `--ignore-engines` flag which is not always recommended but would do it for developing until this plugin is tested and updated to work with node v20

**Comment by mancku** [comment]
Hi @dooglegh @creazy231 I just released the 1.1.20 version of the plugin that fixes this issue. Thanks for telling me about it!
